src/server/hotkey_collector.cpp

```diff
@@ -196,7 +196,8 @@ inline void hotkey_collector::change_state_by_result()
         if (!_result.hot_hash_key.empty()) {
             change_state_to_finished();
             LOG_ERROR_PREFIX("Find the hotkey: {}",
-                             pegasus::utils::c_escape_string(_result.hot_hash_key));
+                             pegasus::utils::c_escape_string(
```

Comment:
Is the c_escape_string always called along with redact_sensitive_string? How about unify to a single function?